traveler-ios-ui/TravelerKitUI/ViewControllers/OrderDetailViewController.swift

```diff
@@ -52,6 +65,14 @@ class OrderDetailViewController: UITableViewController {
         return cell
```

Comment:
Im not sure if its in this PR or the other one, but if you made improvements to product (i.e. like the secondaryDescription you added as an extension), please make sure you make the rest of the code consistent with it. I know its difficult because you dont know, but when a file is touched afterwards or in review it should be changed

Reply:
Oh wait! I think this is addressed in the next PR right?

Reply:
Yep